pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -49,20 +50,25 @@ type crtbLifecycle struct {
 }
 
 func (c *crtbLifecycle) Create(obj *v3.ClusterRoleTemplateBinding) (runtime.Object, error) {
+	logrus.Debugf("%s lifecycle: create", ctrbMGMTController)
```

Comment:
Need this ? I would think we don't. Doesn't look like anything in this file needs to change, besides maybe the misspelling.

Reply:
This is not needed, removing